### ISSUE TITLE
Wording of review redlines in PR 

### ISSUE THREAD

**Issue by SteveLLamb** [SMPTE/html-pub#40]
To make more user friendly to average person, update:

`"Redline to current draft"` to `"Redline to most recent edition"`
and 
`"Redline of PR" ` to `"Redline to Published"`

**Comment by thomasheritage** [comment]
Also: note that the redline is useless (and perhaps shouldn't be shown) on a PR for the initial draft of a document (I committed the template document to `main` before creating a branch (and PR) for the initial draft of the document)

**Comment by thomasheritage** [comment]
> To make more user friendly to average person, update ...

I agree that an update would be very helpful. I'm not entirely clear on the diffs.

Currently we have:
`Redline to current draft`
`Redline to most recent edition`

My understanding is that:
* although the Review links appear near the start of the PR, all the links contain content from the head of the branch (i.e. the head of the branch to be merged by the PR)
* `Redline to current draft` means diff FROM head of `main` branch TO head of the branch to be merged
* `Redline to most recent edition` means diff FROM "latest published edition" TO head of the branch to be merged (where "latest published edition" is given by the `latestEditionTag` property in the .smpte-build.json file)
